### PR TITLE
Mouse zooming

### PR DESCRIPTION
**Pull request by rom1dep**

### CODE REVIEW

**Comment by SSShooter** [comment]
Does this PR include the contents of the previous two PRs? If so, should we consolidate everything into this PR?

**Comment by rom1dep** [comment]
> Does this PR include the contents of the previous two PRs? 

Apparently so, the first commit can be taken out, but the 3rd builds on top of the second. If you see an interest in taking them all eventually, we can use this PR and ignore the other two :)

**Comment by rom1dep** [comment]
I took the chance to fix the formatting/styling issues over https://github.com/SSShooter/mind-elixir-core/pull/277#event-14227445646

File: src/plugin/keypress.ts [hunk 1]
```diff
@@ -178,7 +172,17 @@ export default function (mind: MindElixirInstance) {
       // input
       return
     }
-    const keyHandler = key2func[e.keyCode]
+    const keyHandler = key2func[e.key]
     keyHandler && keyHandler(e)
   }
+
+  mind.map.onwheel = e => {
+    e.preventDefault()
```
Comment (SSShooter) [inline src/plugin/keypress.ts:180]: Maybe we should move the preventDefault() call inside the if condition?

**Review by SSShooter — CHANGES_REQUESTED** [review]

File: src/plugin/keypress.ts (hunk 1 above)
Comment (rom1dep) [reply src/plugin/keypress.ts:180]: Sure, FYI, I only mimicked what's already in place [here for onkeydown](https://github.com/SSShooter/mind-elixir-core/blob/master/src/plugin/keypress.ts#L174).

**Review by SSShooter — APPROVED** [review]

**Comment by SSShooter** [comment]
Thanks for your PRs 😆

**Comment by rom1dep** [comment]
> Thanks for your PRs 😆

Thanks for merging them :)

For the record, your project came to my attention because a PR by @eliandoran recently got merged into https://github.com/TriliumNext/Notes/ , and as such, the next version of the TriliumNext Notes taking app/personal wiki will include a mind mapping feature thanks to mind-elixir!

**Pull request merged**